client/engine/engine.go

```diff
@@ -356,6 +356,23 @@ func (e *Engine) handleObjectiveRequest(or protocols.ObjectiveRequest) (EngineEv
 	e.metrics.RecordObjectiveStarted(objectiveId)
 	switch request := or.(type) {
 
+	// TODO: instead of working with payment channels, we should use standard channel state
+	// and modify LedgerProposals message to work with any kind of channel state (direct or virtual)
```

Comment:
This is not the goal of the sprint, and this is going against the explanation of static embedding.
You may take this approach in your comments if you were to start implementing Dynamic embedding.